packages/freezed/test/integration/single_class_constructor.dart

```diff
@@ -450,7 +450,7 @@ class NullDefault with _$NullDefault {
 class ExplicitConstDefault with _$ExplicitConstDefault {
   factory ExplicitConstDefault(
       //ignore: unnecessary_const
-      [@Default(const <Object>[]) List<Object> value]) = _ExplicitConstDefault;
```

Comment:
Why was this removed? As you can see with the lint, and the name of this class, this is testing support for the explicit const keyword here.

Reply:
Good catch that was the autoformatter removing the const.